include/nighthawk/client/benchmark_client.h

```diff
@@ -59,32 +59,6 @@ class BenchmarkClient {
    * @return bool indicating if latency measurement is enabled.
    */
   virtual bool measureLatencies() const PURE;
-
-  /**
-   * Sets the request method to use when sending request.
-   * @param request_method to set.
-   */
-  virtual void setRequestMethod(envoy::api::v2::core::RequestMethod request_method) PURE;
-
-  /**
-   * Sets request header named 'key' to the specified value.
-   *
-   * @param key Name of the request header to set.
-   * @param value Value to set the request header to.
-   */
-  virtual void setRequestHeader(absl::string_view key, absl::string_view value) PURE;
-
-  /**
-   * @brief Set the Request Body Size
```

Comment:
Consistency wrt @brief

Reply:
Well, we delete that, and checking, with that we are consistent now :-)
This is close to ready for review, working on some final touches, and this needs to be merged with master.